forc-plugins/forc-fmt-v2/Cargo.toml

```
@@ -0,0 +1,21 @@
+[package]
+name = "forc-fmt-v2"
+version = "0.14.4"
```

To avoid having to keep up versioning an unpublished create, just set to 0
```suggestion
version = "0.0.0"
```

---

I left a comment in anticipation of this, but I'll look into it further!

---

Oh shoot I missed that haha